File: src/main/java/baekjoon/Q1008.java
package baekjoon;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Q1008 {
    public static void main(String[] args) {
        // 나눗셈
        divide();
    }

    private static void divide() {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        try {
            String[] strings = br.readLine().split(" ");
            Double a = Double.valueOf(strings[0]);
            Double b = Double.valueOf(strings[1]);
            System.out.println( a / b);
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: baekjoon/q1008/q1008.go
package q1008

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func Q1008() {
	reader := bufio.NewReader(os.Stdin)
	line, _, err := reader.ReadLine()
	if hasErr(err) {
		return
	}

	splitedStrs := strings.Split(string(line), " ")
	fstNum, err := strconv.ParseFloat(splitedStrs[0], 64)
	if hasErr(err) {
		return
	}
	sndNum, err := strconv.ParseFloat(splitedStrs[1], 64)
	if hasErr(err) {
		return
	}

	divide(fstNum, sndNum)

}

func divide(fstNum, sndNum float64) {
	fmt.Println(fstNum / sndNum)
}

func hasErr(err error) bool {
	return err != nil
}
